### DISCUSSION TITLE
开源增长的共识：寻找开源的核心力量 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#113] [Announcements]
# 开源增长的共识：寻找开源的核心力量 - 「开源之道」

https://opensourceway.community/posts/open-source-economic/open-source-growth-common-sense/

**Comment by lijiangsheng1** [comment]
0.  摒弃利他主义，坚持互惠和博弈的合作精神。
1.  开源项目被现实所需要，无论是商业诉求，还是人类基础设施。
2.  匠人精神，能够为了将事情做好而去做好。
3.  社会关怀，能够有渠道接受社会捐赠、捐助、赞助。
4.  知识财产权的观念成为社会常识，开源许可是具有约束力和激励作用的
5.  职业共同体的伦理、精神和文化的遵循、尊重、自尊
6.  权利的让渡是数字化世界的充分必要条件
7.  惩罚搭便车者的社会偏好
8.  开放透明的交流是日常的必须
9.  和收取许可费用的闭源软件形成竞争-合作关系，同为软件生态下的不同方式

**Reply by lijiangsheng1** [reply]
制度安排：正式法律法规、非正式共同体伦理与规范、文化价值观等共同塑造的社会体系。

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
俺的直觉, 4~7条就足够了, 多于9条就必定有重复或是凑数了...

0: Why not 为第一动力;
1: 每个参与的人, 第一目标不是为了赚钱养家;
2: 没有任何形式的 KPI, 但是, 引导新人进入, 或是吸引越来越多的人认同并参与是公认的乐事儿
3: 比起版本稳定增长, 每次解决一个具体问题引发的各种知识/技巧/经验的增长, 更加快乐, 并乐于将之第一时间记录在案, 通过文章或是文档形式公诸于世;
...

**Reply by lijiangsheng1** [reply]
自由人～ 乃核心也。

**Comment by JiaoXianjun** [comment]
感觉现在时代变了。现在开源增长和成功核心要素是金主爸爸，即来自大型公司和组织的开源项目以及贡献者。这种开源活动（大多）不背负（直接的）盈利和商业化压力，因此更容易践行一些高大上的冠冕堂皇的占领道德制高点的开源理念和文化，最后呈现出一种：看这些高大上的根正苗红的开源理念会导致成功！

**Reply by lijiangsheng1** [reply]
商业公司的赞助非常重要～  但不是唯一的增长路径。
